src/main/test/UserTest.java

```diff
@@ -0,0 +1,53 @@
+package entities;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+
+public class UserTest{
+    @Test
+    public void testIsAdmin() {
+        User user =  new User(true, 0);
+        boolean actual = user.isAdmin();
+        boolean expected = true;
+
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    public void testConstructerFavourites() {
+        User user = new User(false, 0);
+        boolean actual = user.getFavourites;
+        boolean expected = true;
+
```

Comment:
user.getFavourites should return a list. get.favourites needs ()

Reply:
Right good catch